apis/v1beta1/object_reference_types.go

```diff
@@ -99,9 +99,22 @@ type BackendObjectReference struct {
 	// +kubebuilder:default=""
 	Group *Group `json:"group,omitempty"`
 
-	// Kind is kind of the referent. For example "HTTPRoute" or "Service".
+	// Kind is the Kubernetes resource kind of the referent. For example
+	// "Service" or "HTTPRoute".
```

Comment:
I know that HTTPRoute is not something that was added by you, but reading it now it really doesn't make sense that HTTPRoute could be a BackendRef unless we had defined route delegation.
```suggestion
	// "Service".
```

Reply:
Sounds good, I had a similar thought. Changed in bef0c2087ba2c5e69b69a5689a9c7c1799b32b86 :+1: